.claude/agents/review/tech-lead.md

```
@@ -7,196 +7,138 @@ model: opus
-
-## 문서 출력 규격
+## Output Format
 
 ### 개별 리뷰 문서 (`docs/review/{분야}-review.md`)
 
 ```markdown
 # {분야} Review — Forgather Backend
 
 > 리뷰 일시: {날짜}
-> 리뷰 범위: {분석 대상 파일/디렉토리}
+> 리뷰 범위: {분석 대상}
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)